airflow/contrib/operators/gcp_spanner_operator.py

```diff
@@ -0,0 +1,132 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from airflow import AirflowException
+from airflow.contrib.hooks.gcp_spanner_hook import CloudSpannerHook
+from airflow.models import BaseOperator
+from airflow.utils.decorators import apply_defaults
+
+
+class CloudSpannerInstanceDeployOperator(BaseOperator):
+    """
+    Creates a new Cloud Spanner instance or, if an instance with the same name exists,
```

Comment:
Can you clarify in doc that which field is checked for existence?
"if an instance with the same instance_id exists"
"if an instance with the same project_id and instance_id exists"

Reply:
Done